src/main/java/com/scalar/database/api/TransactionState.java

```diff
@@ -1,6 +1,6 @@
 package com.scalar.database.api;
 
-/** */
+/** Flag used to represent either the state of a record in a transaction or a transaction itself. */
```

Comment:
What is the relationship between the state of a record and transaction state ?
What is says is not false but I think it is not very informative and probably misleading.